compiler/compiler.cpp

```diff
@@ -248,6 +250,7 @@ bool compiler_execute(CompilerSettings *settings) {
     >> PipeC<CalcEmptyFunctions>{}
     >> PassC<CalcActualCallsEdgesPass>{}
     >> SyncC<FilterOnlyActuallyUsedFunctionsF>{}
+    >> PipeC<InitCppDestDir>{}
```

Comment:
Is it true, that loading mtime will start here and run in parallel until the next sync pipe (CFGBeginSync) waits for it?
I mean, that if a hard disk is slow, the execution point won't pass CFGBeginSync (it will hang there waiting for mtime)?

Reply:
yes

Reply:
(outdated)